modules/slugs/admin/slugs-admin.php

```diff
@@ -143,38 +139,36 @@ function qts_draw_meta_box( $post ) {
 }
 
 /**
- * Sanitize title as slug, if empty slug.
+ * Sanitize a post slug for a given language.
  *
- * @param $post (object) the post object
- * @param $slug (string) the slug name
- * @param $lang (string) the language
+ * @param string $slug slug name
+ * @param WP_Post $post the post object
+ * @param string $lang the language
  *
- * @return string the slug validated
+ * @return string sanitized slug
  */
-function qts_validate_post_slug( $slug, $post, $lang ) {
+function qts_sanitize_post_slug( $slug, $post, $lang ) {
     $post_title = trim( qtranxf_use( $lang, $post->post_title ) );
     $post_name  = get_post_meta( $post->ID, QTS_META_PREFIX . $lang, true );
     if ( ! $post_name ) {
         $post_name = $post->post_name;
     }
 
     //TODO: if has a slug, test and use it
-    //TODO: and then replace the default slug with the dafault language slug
-    $name = ( $post_title === '' ) ? $post_name : $post_title;
-
+    //TODO: and then replace the default slug with the default language slug
+    $name = empty( $post_title ) ? $post_name : $post_title;
     $slug = trim( $slug );
-
-    $slug = ( $slug === '' ) ? sanitize_title( $name ) : sanitize_title( $slug );
+    $slug = empty( $slug ) ? sanitize_title( $name ) : sanitize_title( $slug );
```

Comment:
This is not covering values of `$slug` that would be evaluated as `false`, like `'0'`. Better the previous test.

Reply:
Reverted (though `false` should never happen).